### ISSUE TITLE
Documentation Addition

### ISSUE THREAD

**Issue by alshapton** [abnerjacobsen/tinydb-jsonorm#2]
Is there any intention to modify/extend the documentation for this project in the near future......?

**Comment by abnerjacobsen** [comment]
@alshapton Yes, I plan to write some documentation and examples, but I'm finalizing the project that motivated me to write tinydb-jsonorm  and I can only have time to write the documentation in about 30 days. As additional information: The project where I'm using tinydb-jsonorm is [Daspanel](https://github.com/daspanel/daspanel) and in these sources you can get an idea of how to use it:

Model: https://github.com/daspanel/api-server/blob/master/api_server/modules/sites/models/tinyj.py
Usage of the model: https://github.com/daspanel/api-server/blob/master/api_server/modules/sites/api.py

**Comment by alshapton** [comment]
Thanks - I think that daspanel is looking good! I'll keep an eye out for documentation and so on.... Meanwhile I'll list your project on TinyDB's list of active projects.
